streams/shared/src/main/scala/zio/stream/ZSink.scala

```diff
@@ -508,29 +503,29 @@ trait ZSink[-R, +E, +A0, -A, +B] { self =>
    * Right: ===== SUCCEEDS!
    *             xxxxxxxxx <- Should NOT be consumed
    */
-  final def orElse[R1 <: R, E1, A00 >: A0, A1 <: A, C](
-    that: ZSink[R1, E1, A00, A1, C]
-  )(implicit ev: A1 =:= A00): ZSink[R1, E1, A00, A1, Either[B, C]] =
-    new ZSink[R1, E1, A00, A1, Either[B, C]] {
+  final def orElse[R1 <: R, E1, C](
```

Comment:
Feels good to see the complex combinators migrated without too much hassle. It looks like we got them right last time. :smile: